src/api/procedures/createAsset.ts

```diff
@@ -169,60 +174,142 @@ export async function prepareCreateAsset(
   const rawFundingRound = optionize(stringToBytes)(fundingRound, context);
   const rawDisableIu = booleanToBool(!requireInvestorUniqueness, context);
 
-  let fee: undefined | BigNumber;
-
-  const rawTicker = stringToTicker(ticker, context);
+  const newAsset = new Asset({ ticker }, context);
 
-  // we waive any protocol fees if the Asset is created in Ethereum. If not created and ticker is not yet reserved, we set the fee to the sum of protocol fees for ticker registration and Asset creation.
+  let fee: undefined | BigNumber;
 
+  /*
+   * we waive any protocol fees if the Asset is created in Ethereum. If not created and ticker is not yet reserved,
+   *   we set the fee to the sum of protocol fees for ticker registration and Asset creation.
+   */
   const classicTicker = await asset.classicTickers(rawTicker);
   const assetCreatedInEthereum =
     classicTicker.isSome && boolToBoolean(classicTicker.unwrap().isCreated);
 
   if (assetCreatedInEthereum) {
     fee = new BigNumber(0);
   } else if (status === TickerReservationStatus.Free) {
-    const [registerTickerFee, createAssetFee] = await Promise.all([
-      context.getProtocolFees({ tag: TxTags.asset.RegisterTicker }),
-      context.getProtocolFees({ tag: TxTags.asset.CreateAsset }),
-    ]);
-    fee = registerTickerFee.plus(createAssetFee);
+    fee = await addManualFees(
+      new BigNumber(0),
+      [TxTags.asset.RegisterTicker, TxTags.asset.CreateAsset],
```

Comment:
Why is it adding `RegisterTicker` fee? What does it happen if ticker was registered previously?